index c72fd9d..b827d8e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,7 +64,7 @@ INCLUDE( "${ILCUTIL_ROOT}/cmakemodules/MacroRootDict.cmake" )
 SET( lib_input_dirs gen kern lctpc/gearTPC )
 
 FOREACH( lib_input_dir ${lib_input_dirs} )
-    LIST( APPEND ROOT_DICT_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/${lib_input_dir} )
+    LIST( APPEND ROOT_DICT_INCLUDE_DIRS /usr/include/kaldet/${lib_input_dir} )
 ENDFOREACH()
 
 LIST( APPEND ROOT_DICT_INCLUDE_DIRS ${KalTest_INCLUDE_DIRS} )
@@ -77,7 +77,7 @@ FOREACH( lib_input_dir ${lib_input_dirs} )
 
     AUX_SOURCE_DIRECTORY( ${lib_input_dir} lib_sources )
 
-    PREPARE_ROOT_DICT_HEADERS( ${lib_input_dir} )
+    PREPARE_ROOT_DICT_HEADERS( /usr/include/kaldet/${lib_input_dir} )
 
     INSTALL_DIRECTORY( ${lib_input_dir}/ DESTINATION "include/kaldet"
         FILES_MATCHING PATTERN "*.h" PATTERN "LinkDef.h" EXCLUDE

