diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index ec172c6..10c2d85 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -3,7 +3,7 @@
 SET( lib_input_dirs geomlib kallib kaltracklib utils bfield )
 
 FOREACH( lib_input_dir ${lib_input_dirs} )
-    LIST( APPEND ROOT_DICT_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/${lib_input_dir} )
+    LIST( APPEND ROOT_DICT_INCLUDE_DIRS /usr/include/kaltest/${lib_input_dir} )
 ENDFOREACH()
 
 #MESSAGE( STATUS "ROOT_DICT_INCLUDE_DIRS: ${ROOT_DICT_INCLUDE_DIRS}" )
